src/products/README.md

```diff
@@ -40,3 +40,39 @@ Updates the current inventory value for the product identified by `{productID}`.
 Returns details on all categories.
 ### GET /categories/id/{categoryID}
 Returns details on the category identified by `{categoryID}`.
+
+## Testing
+To run integration tests for the Products service a Python virtual environment is required. You must have Python 3.8+ installed on your system to run the commands below. The commands are written to be ran from the test directory of the products service (`src/products/test`).
+
+The following command will create a virtual environment. 
+```console
+python3 -m venv .venv
+```
+
+Some environment variables are required to run the tests and need to be added to the virtual environment. The example below will work for local development. Change as required depending on environment.
+```console
```

Comment:
reference the new CloudFormation output maybe ?

Reply:
Can you clarify which part of the output should be referenced here? As the tests are separate I’m not sure it's relevant.

Reply:
referring to


```sh
aws cloudformation describe-stacks --stack-name retaildemostore \
  --region REGION \
  --query "Stacks[0].Outputs[?OutputKey=='ExportEnvVarScript'].OutputValue" 
  --output text
```

in src/README.md

Reply:
I think the only relevant variables there are the service URLs for the deployment so I believe we can leave this out or just point the user in the correct direction of the service URLs only rather than showing everything?